components/zcash_address/LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2021 Electric Coin Company
```

Comment:
Electric Coin Company, or "Zcash developers"?

Reply:
ECC (matches the license text in `zcash_primitives/LICENSE-MIT` which is where the F4Jumble impl was moved from, and also matches ECC having copyright assignment for the rest of the code I've written in this module).

Reply:
OK, but it'll need to be changed as soon as we get any external contributions, unless we intend to require copyright assignment (which we *don't* do for any of our other MIT/Apache code).

Reply:
We've been accepting external contributions for quite some time with the rest of the repo licensed in this way. If there's a change that needs making, it should be made uniformly.

Reply:
I think this is violating the contributor's copyright, strictly speaking. Filed https://github.com/zcash/librustzcash/issues/393 .